code/src/Components/CheckboxInput.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+const MemberConfimation = ["Yes", "No"]
+
+export const CheckboxInput = ({ membership, onMembershipInputChange, Counter}) => {
+
+  return (
+    <form className="member-form">
+      <h3>Are you already a member of our art assocation <span className="highlight">Art Emergence?</span></h3>
+      {MemberConfimation.map(answer => (
+        <label 
+          htmlFor="membership" 
+          aria-label="membership"
+          key={answer}>
+          <input
+          type="checkbox"
+          required
```

Comment:
I don't this "require" works, because "require" is a built-in HTML5 form validation. I think an alterntive is to have a form element around the button and a button with type=submit.